### PR TITLE
Milestone 1:Project setup for Tic Tac Toe

### PR DESCRIPTION
**Pull request by evansinho**
#### In this milestone :
- we created starter files for the project

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+puts 'Hello world!'
```
Comment (euqueme) [inline bin/main.rb:3]: - [ ] `main.rb` is [executable file](https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/):
* `#!/usr/bin/env ruby` is the top line
* puts "Hello world" in the following line ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/execute_bin.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

Please add the top line

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @evansinho and @rajkumardas2701 :balloon: :lollipop: 

### Good job setting up your project, claps to you for using a template for your Readme in this milestone :clap: :clap: :clap: there are a few changes to make so keep reading and take a look at your code below :point_down: 

#### Project-specific

- [ ] It has `lib` directory (with .gitkeep file OR any other **empty** file(s)). ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/lib_dir_exist.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

- Please create the lib directory and make sure is not in the .gitignore file so it gets uploaded to your repository

### That's it, team! :raised_hands: 

### Please :pray:  do these changes :man_technologist: :man_technologist:  and submit back for a code review when you are done :memo: 

Happy Coding! :rocket: 

Any question? please ask me on slack **@ Maru** :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :vertical_traffic_light:

File: bin/main.rb (hunk 1 above)
Comment (evansinho) [reply bin/main.rb:3]: there is lib folder in our folder structure and it is executable using the `bin/main.rb` command on bash. please kindly check again.

**Review by pelzolga123 — APPROVED** [review]
Approved.

**Comment by pelzolga123** [comment]
## Great work, Congratulation :+1: Project Approved.
You did all changes required previously. Go ahead and merge to development.
_Goodluck in your next milestone_ *Happy coding*

**Pull request merged**